lib/actions/ui.js

```diff
@@ -87,12 +219,13 @@ export function matchContentToUrl (location) {
         }
         break
       case 'start':
-      case '@':
+      case '@': {
         // Parse comma separated params (ensuring numbers are parsed correctly).
-        let [lat, lon, zoom, routerId] = id ? idToParams(id) : []
+        const [lat, lon, zoom, routerId] = id ? idToParams(id) : []
```

Comment:
This change is causing a11y tests to fail. It needs to be changed back to `let`

Reply:
changed in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/a6de95114aed8b19ec7c8910f2e5e500a88a524d